app/views/api/clidocs/casestudy_metagenomics.html.erb

```diff
@@ -1,121 +1,434 @@
+<!--   -->>read1/1|2
+<!--   -->SRTRPMPQKRLTAVGRAAARWSQISATSALRVG
+<!--   -->>read1/1|3
+<!--   -->RARGRCPRRD*QRWGGRRRGGRRSRRPLR*GS
+<!--   -->>read1/1|1R
+<!--   -->PDPQRRGRRDLRPPRRRPPHRCQSLLGHRPRAR
+<!--   -->>read1/1|2R
+<!--   -->PTLNAEVAEICDHLAAARPTAVNLFWGIGRVRD
+<!--   -->>read1/1|3R
+<!--   -->RPSTQRSPRSATTSPPPAPPLSISSGASAACA
+<!--   -->>read1/2|1
+<!--   -->KMATG**SRGRPPSTIPQ*SITNHDAQRLAVRA
+<!--   -->>read1/2|2
+<!--   -->RWRLDDDRAAGHHLRSRSEALLTMMRNDSLSEL
+<!--   -->>read1/2|3
+<!--   -->DGDWMMIARQATIYDPAVKHY*P*CATTRCQS
+<!--   -->>read1/2|1R
+<!--   -->*L*QRVVAHHG**CFTAGS*MVACRAIIIQSPS
+<!--   -->>read1/2|2R
+<!--   -->SSDSESLRIMVSNASLRDRRWWPAARSSSSRHL
+<!--   -->>read1/2|3R
+<!--   -->ALTASRCASWLVMLHCGIVDGGLPRDHHPVAI
+<!--   -->>read2/1|1
+<!--   -->RLLVRVLCRASSSRTAHPMRTRHSRYAVRAPRF
+<!--   -->>read2/1|2
+<!--   -->DCWCGCSAGLLHLGPRIRCAHGTVGMR*EHLAF
+<!--   -->>read2/1|3
+<!--   -->IAGAGALPGFFISDRASDAHTAQ*VCGESTSL
+<!--   -->>read2/1|1R
+<!--   -->KSEVLSPHTYCAVCASDARSEMKKPGRAPAPAI
+<!--   -->>read2/1|2R
+<!--   -->KARCSHRIPTVPCAHRMRGPR*RSPAEHPHQQS
+<!--   -->>read2/1|3R
+<!--   -->KRGALTAYLLCRVRIGCAVRDEEARQSTRTSN
+<!--   -->>read2/2|1
+<!--   -->RNLAAHRDQIGIVRCVTRAQVICPYPQYIQLCF
+<!--   -->>read2/2|2
+<!--   -->AILRRTAIKSG*SGV*RVLRSSARTRNTFNCVL
+<!--   -->>read2/2|3
+<!--   -->QSCGAPRSNRDSPVCNACSGHLPVPAIHSTVF
+<!--   -->>read2/2|1R
+<!--   -->*NTVECIAGTGR*PEHALHTGLSRFDRGAPQDC
+<!--   -->>read2/2|2R
+<!--   -->KTQLNVLRVRADDLSTRYTPDYPDLIAVRRKIA
+<!--   -->>read2/2|3R
+<!--   -->KHS*MYCGYGQMT*ARVTHRTIPI*SRCAARL
+<!--   -->>read3/1|1
+<!--   -->PSSPIELSYCPTSARSITESSSTISPMASM*TP
+<!--   -->>read3/1|2
+<!--   -->PQARSNFPTARPQRGALPNRRQQSPRWLLCELH
+<!--   -->>read3/1|3
+<!--   -->LKPDRTFLLPDLSAEHYRIVVNNLPDGFYVNS
+<!--   -->>read3/1|1R
+<!--   -->MEFT*KPSGRLLTTIR*CSALRSGSRKVRSGLR
+<!--   -->>read3/1|2R
+<!--   -->WSSHRSHRGDC*RRFGNAPR*GRAVGKFDRA*G
+<!--   -->>read3/1|3R
+<!--   -->GVHIEAIGEIVDDDSVMLRAEVGQ*ESSIGLE
+<!--   -->>read3/2|1
+<!--   -->QNGVGATAP*AGSVGGIGACARGC*WRCPWIPE
+<!--   -->>read3/2|2
+<!--   -->KMEWALLLREQGQLEGLGRALGDANGGAPGFLR
+<!--   -->>read3/2|3
+<!--   -->KWSGRYCSVSRVSWRDWGVRSGMLMAVPLDS*
+<!--   -->>read3/2|1R
+<!--   -->SQESRGTAISIPERTPQSLQLTLLTEQ*RPLHF
+<!--   -->>read3/2|2R
+<!--   -->LRNPGAPPLASPSARPNPSN*PCSRSSSAHSIL
+<!--   -->>read3/2|3R
+<!--   -->SGIQGHRH*HPRAHAPIPPTDPAHGAVAPTPF
+        </pre>
+      </div>
+    </div>
+
+    <div class="card">
+      <div class="card-title card-title-colored"><h2 class="card-title-text">Taxonomic Identification</h2></div>
+      <div class="card-supporting-text">
         <p>
-          Then we can compute the biodiversity distribution of a sample by counting the number of occurrences of each taxon <i>(timing: 4.8 secs)</i>.
-    </p>
+          UMGAP uses exact substring matching to identify reads.
+          The next step of the pipeline fragments each amino acid sequence into 9-mers,
+          all of its substrings of length 9, and looks them up in our index.
```

Comment:
`The next step of the pipeline fragments each amino acid sequence into 9-mers. The 9-mers of a text are all possible substrings of length 9. For example, all 9-mers of the string "ABCDEFGHIJK" are "ABCDEFGHI", "BCDEFGHIJ" and "CDEFGHIJK".`

_[Original comment](https://github.ugent.be/unipept/unipept/pull/642#discussion_r16553) by @bmesuere on Mon Aug 20 2018 at 13:57._